am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessMetadata.java

```diff
@@ -23,6 +23,9 @@
     private final AccessorType accessorType;
     @NotNull
     private final JsonPointer attribute;
+    @NotNull
+    private final SecurityClassification securityClassification;
```

Comment:
There should be no changes for security classification on the PR. What is more SC got removed from am table.

Reply:
👍